meson.build

```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: LGPL-2.1 OR BSD-2-Clause
+project('libbpf', 'c',
+        version : '0.0.3',
+        license : 'LGPL-2.1 OR BSD-2-Clause',
+        default_options : [
+                'prefix=/usr',
+        ],
+        meson_version : '>= 0.46',
+        )
+
+patchlevel = '0'
+extraversion = '3'
```

Comment:
Nit: Let's reduce number of places that should be kept in sync. Version components can be specified once and then used in `project.version` to make full version, [same way as in `src/Makefile`](https://github.com/libbpf/libbpf/blob/8bbb21c227d64efe3ad96cfd427984e51f6ee549/src/Makefile#L3-L7).

Reply:
@rdna 

Thanks, fixed. Splitting `version` string instead of joining due to
`ERROR: First statement must be a call to project`
Duplications are gone.